frontend/src/pages/PropertyDetails.jsx

```
@@ -0,0 +1,813 @@
+                            }}
+                            className="absolute left-0 z-10 bg-white/20 cursor-pointer hover:bg-white/40 p-4 rounded-full transition-all duration-300 text-white"
+                        >
+                            <ChevronLeft size={24} />
+                        </button>
+                    )}
+
+                    {/* Image Container */}
+                    <div className="relative w-full h-full flex items-center justify-center p-8">
+                        <img 
+                            src={`http://localhost:4000/${images[currentImageIndex]}`} 
+                            alt={`Full screen image ${currentImageIndex + 1}`}
```

Biome (1.9.4)

[error] Avoid the words "image", "picture", or "photo" in img element alt text.

Screen readers announce img elements as "images", so it is not necessary to redeclare this in alternative text.

(lint/a11y/noRedundantAlt)